FeedStoreChallenge/CD_Cache.swift

```diff
@@ -0,0 +1,47 @@
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CD_Cache)
+public class CD_Cache: NSManagedObject {}
```

Comment:
`CD_Cache` is an internal implementation detail of `CoreDataFeedStore`, thus it should not be public.
Also this class is not intended to be inherited thus it should be `final`
Please remove all `public` modifiers from this class declaration and its properties and methods and make the class `final`.
```suggestion
final class CD_Cache: NSManagedObject {}
```

Suggestion:
Having underscores in the name of a class is not a common thing in swift. Usually classes are named in camel case starting from a capital letter. 
```suggestion
final class CDCache: NSManagedObject {}
```